SETUP.md

```diff
@@ -0,0 +1,95 @@
+# AutoSpotting Setup #
+
+It's relatively easy to build and install your own version of this tool's binaries, 
+removing your dependency on the author's version, and allowing any customizations
+and improvements your organization needs. You'll need to set up a local environment
+to run Go, compile the binaries locally, upload them to an S3 bucket in your AWS
+account, and update the CloudFormation stack to use those new binaries.
+
+## Compiling the binaries locally ##
+
+1. Install Go via the binaries [here](https://golang.org/dl/) or Homebrew:
+
+   `brew install go`
+2. Verify that Go is installed
+
+   `go version`
+   
+3. Set up a directory for your Go development. I'm using `godev` in my home directory
+for this example.
+
+4. Set the `GOPATH` environment variable to point at your `godev` directory:
+
+   `export GOPATH=$HOME/godev`
+   
+   Optionally add this line to your .bash_profile to persist across console sessions.
+
+5. Navigate to your `godev` directory and run the following to bring in the AutoSpotting 
+   project:
+   
+   `go get github.com/cristim/autospotting`
+   
+   This will download the source from GitHub as well as pull in any necessary 
+   dependencies.
+
+6. Navigate to the root of the AutoSpotting repository:
+
+   `cd src/github.com/cristim/autospotting`
+
+7. Try building and running the tests to make sure everything pulled in correctly. More 
+   details on the available directives below.
+
+   `make install`
+
+   `make test`
+
+8. (Optional) You may want to make a minor change to the source code so you can tell when
+   the tool is running your own custom-built version. If so, add a line like this to the 
+   `autospotting.go` file's `main()` function:
+
+   `fmt.Println("Running <my organization name> binaries")`
+
+## Using your own binaries in AWS ##
+
+1. Set up an S3 bucket in AWS that will host the binaries
+
+2. The Makefile contains a `BUCKET_NAME` variable that tells it where to upload new
+   binaries. Edit the Makefile and change it from the default `cloudprowess` to the name 
```

Comment:
I think the default has been changed to `cloudprowess-private` - mentioned in various places afterwards as well